src/WebJobs.Script/Workers/ProcessManagement/WorkerProcessCountOptions.cs

```diff
@@ -7,6 +7,8 @@ namespace Microsoft.Azure.WebJobs.Script.Workers
 {
     public class WorkerProcessCountOptions
     {
```

Comment:
rename :)

Reply:
Punting for a future PR -- also this may be a breaking change depending on how we parse the worker options json file.